BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -665,7 +672,9 @@ def sign_component (args):
     auth_dict = {
         'none'    : 'NONE',
         'sha256'  : 'SHA2_256',
+        'sha384'  : 'SHA2_384',
         'rsa2048' : 'RSA2048',
+        'rsa3072' : 'RSA3072',

```

Comment:
Since it is the almost the same string, do we need dict ? Can we just let the input argument use NONE, SHA2_256, ... directly?